delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/StoragesConfig.scala

```diff
@@ -206,7 +210,8 @@ object StoragesConfig {
       defaultWritePermission: Permission,
       showLocation: Boolean,
       defaultMaxFileSize: Long,
-      defaultBucket: String
+      defaultBucket: String,
+      prefix: Uri
```

Comment:
It would be better to have an option here

Reply:
Agreed